movo:
  hardware_interface:
     loop_hz: 100
     joints:
       - right_shoulder_pan_joint
       - right_shoulder_lift_joint
       - right_arm_half_joint
       - right_elbow_joint
       - right_wrist_spherical_1_joint
       - right_wrist_spherical_2_joint
       - right_wrist_3_joint
       - right_gripper_finger1_joint
       - left_shoulder_pan_joint
       - left_shoulder_lift_joint
       - left_arm_half_joint
       - left_elbow_joint
       - left_wrist_spherical_1_joint
       - left_wrist_spherical_2_joint
       - left_wrist_3_joint
       - left_gripper_finger1_joint
       - linear_joint
       - pan_joint
       - tilt_joint
